lambdas-py/.env.integration

```
@@ -0,0 +1,17 @@
+TEST_USER_ID=cmiio2mkv0000l1049q0gnft3
+TEST_STEAM_ID=76561198026626729
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TEST_STEAM_ID key should go before the TEST_USER_ID key

(UnorderedKey)